### PR TITLE
Fix removing of columns via mouse against release/4.1

### PR DESCRIPTION
**Pull request by knight-bubble**
#2780

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://hirtie-maxim-fix-19432-remove-columns-2.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://hirtie-maxim-fix-19432-remove-columns-2.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

**Comment by knight-bubble** [comment]
@floriank, could you please check this PR?

**Comment by 0xF013** [comment]
works for me except one small issue: if, while dragging, you hover the anchor, the drag fails.

**Comment by floriank** [comment]
Original WP was http://community.openproject.org/work_packages/19432

**Comment by floriank** [comment]
I restarted the travis job which failed.

**Comment by 0xF013** [comment]
the drag over the 'x' seems to persist but I am no longer sure it's entirely our fault. We should extract this to a new WP and merge this. @myabc @floriank thoughts?

**Comment by knight-bubble** [comment]
@floriank, Could reproduce the bug from #2780, but couldn't find a solution for it. This issue is reproducable for ui-select too. So I can say that the problem is in ui-select.

**Comment by floriank** [comment]
I am currently trying to reproduce..

**Comment by 0xF013** [comment]
> I am currently trying to reproduce

wow isn't that dirty

**Comment by floriank** [comment]
I just decided that i want to have your babies. @0xF013

File: frontend/app/work_packages/controllers/dialogs/columns.js [hunk 1]
```diff
@@ -100,4 +102,12 @@ module.exports = function($scope, $filter, columnsModal, QueryService, WorkPacka
 
     vm.unusedColumns = _.filter(vm.availableColumns, isUnused);
   };
+  
+  //hack to prevent dragging of close icons
+  $timeout(function(){
+    jQuery('.columns-modal-content .ui-select-match-close')
```
Comment (floriank) [inline frontend/app/work_packages/controllers/dialogs/columns.js]: Ha! Please use `angular.element`!

**Comment by floriank** [comment]
Also, remove the WIP label if this is mergable.

**Pull request merged**